src/drivers/bbblue_adc/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ if ("${BOARD}" STREQUAL "bbblue")
 			#
 			# so include bbblue board specific code here:
 			../boards/bbblue/init.c
+			../boards/bbblue/pwr.cpp
```

Comment:
This is pretty bad cmake. These files from ../boards/bblue should be built by cmake in the relevant folder and linked.